_commands/tools/units.md

```diff
@@ -0,0 +1,79 @@
+---
+---
+
+units
+--
+
+`units` gives conversion rates from one scale to another. 
+
+~~~ bash
+$ units [-f unitsfile] [-q] [-v] [from-unit to-unit]...
+~~~
+
+<!--more-->
+
+The command above will list the value you will need to multiply (*) or divide (/) your original number by to convert to your desired units listed.
+
+The units are defined in an external data file. You can use the extensive data file that comes with this program, or you can provide your own data file to suit your needs. You can use the program interactively with prompts, or you can use it from the command line.
+
+~~~ bash
+$ units
+~~~The command above will list the value you will need to multiply (*) or divide (/) your original number by to convert to your desired units listed.
```

Comment:
This needs to be moved to a newline; it's not rendering correctly. You also have the same line twice. It is also on `15`.

![image](https://cloud.githubusercontent.com/assets/5882053/20082520/641b4ef8-a524-11e6-8d3d-6fd4661f2cab.png)